scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -156,12 +170,18 @@ task ExtractVariantAnnotations {
 
 		Int memory_mb = 28000
 		Int command_mem = memory_mb - 1000
+		File? monitoring_script
 	}
 	Int disk_size = ceil(size(input_vcf, "GB") + size(input_vcf_index, "GB") + 100)
 
+
 	command {
 		set -e
 
+		if [ -s ~{monitoring_script} ]; then
+			bash ~{monitoring_script} > monitoring.log &
+		fi
```

Comment:
If this evaluates to false there won't be a `monitoring.log` but the output is not optional